include/vsg/core/type_name.h:31:48: error: 'int8_t' was not declared in this scope

**Describe the bug**
The build of VulkanSceneGraph fails with gcc13 for MinGW.

**To Reproduce**
Steps to reproduce the behavior:
1. see appended log file
```
In file included from /home/abuild/rpmbuild/BUILD/VulkanSceneGraph-1.1.0+20240307+git.148340c/include/vsg/maths/vec2.h:26,
[  103s]                  from /home/abuild/rpmbuild/BUILD/VulkanSceneGraph-1.1.0+20240307+git.148340c/include/vsg/maths/vec3.h:26,
[  103s]                  from /home/abuild/rpmbuild/BUILD/VulkanSceneGraph-1.1.0+20240307+git.148340c/include/vsg/maths/sphere.h:26,
[  103s]                  from /home/abuild/rpmbuild/BUILD/VulkanSceneGraph-1.1.0+20240307+git.148340c/include/vsg/maths/plane.h:26,
[  103s]                  from /home/abuild/rpmbuild/BUILD/VulkanSceneGraph-1.1.0+20240307+git.148340c/include/vsg/maths/mat4.h:15,
[  103s]                  from /home/abuild/rpmbuild/BUILD/VulkanSceneGraph-1.1.0+20240307+git.148340c/include/vsg/animation/Joint.h:15,
[  103s]                  from /home/abuild/rpmbuild/BUILD/VulkanSceneGraph-1.1.0+20240307+git.148340c/src/vsg/animation/Joint.cpp:13:
[  103s] /home/abuild/rpmbuild/BUILD/VulkanSceneGraph-1.1.0+20240307+git.148340c/include/vsg/core/type_name.h:31:48: error: 'int8_t' was not declared in this scope
[  103s]    31 |     template<> constexpr const char* type_name<int8_t>() noexcept { return "int8_t"; }
[  103s]       |                                                ^~~~~~
[  103s] /home/abuild/rpmbuild/BUILD/VulkanSceneGraph-1.1.0+20240307+git.148340c/include/vsg/core/type_name.h:17:1: note: 'int8_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
[  103s]    16 | #include <typeinfo>
[  103s]   +++ |+#include <cstdint>
[  103s]    17 | 
``
**Expected behavior**
The build should not fail

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: Linux openSUSE 
 - Version Tumbleweed

**Additional context**
[1.log.txt](https://github.com/vsg-dev/VulkanSceneGraph/files/14564503/1.log.txt)

--- a/include/vsg/core/type_name.h
+++ b/include/vsg/core/type_name.h
@@ -12,6 +12,7 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 
 </editor-fold> */
 
+#include <cstdint>
 #include <string>
 #include <typeinfo>
 

